javascript/javascript1/week1/dogage.js

```diff
@@ -0,0 +1,13 @@
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let dogYear = dogYearFuture-dogYearOfBirth;
+
+let shouldShowResultInDogYears = false;
```

Comment:
`const` instead of `let`

Reply:
left thi variable "shouldShowResultInDogYears" as let because I ws not sure if I can use 'const' to boolean